test/integration/users.test.js

```diff
@@ -261,6 +261,31 @@ describe("Users", function () {
           return done();
         });
     });
+
+    it("Should return 400 for invalid Linkedin ID", function (done) {
+      chai
+        .request(app)
+        .patch("/users/self")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .send({
+          linkedin_id: "invalid@linkedin_id",
+        })
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(400);
+          expect(res.body).to.be.an("object");
+          expect(res.body).to.eql({
+            statusCode: 400,
+            error: "Bad Request",
+            message: "Invalid Linkedin ID. ID should not contain special character @",
+          });
+
+          return done();
+        });
+    });
```

Comment:
I seen in acceptance criteria 
<img width="686" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/107163260/c8937f23-f45f-475a-ad07-8a3b71bf4de9">
No Spaces: Most platforms do not allow spaces in social id, so make sure your social id(LinkedIn, Instagram) is a single word or utilizes underscores or hyphens to separate words.

Can you write test for this above

Reply:
Done! , @shubhamsigdar1 You can check now, added validation for space as well

Reply:
Done @shubhamsigdar1, Written unit tests and integration test for space error.